deny.toml

```diff
@@ -1,12 +1,7 @@
 # This file defines configuration for the cargo deny command
 # Ref: https://github.com/EmbarkStudios/cargo-deny
-targets = []
```

Comment:
Was there an issue with the deny.toml throwing errors? We want to keep this configuration.

Reply:
Oops, it was deprecated but I chose the wrong solution. I think I fixed it now